dspace-server-webapp/src/test/java/org/dspace/app/rest/CCLicenseRemovePatchOperationIT.java

```diff
@@ -0,0 +1,100 @@
+ */
+public class CCLicenseRemovePatchOperationIT extends AbstractControllerIntegrationTest {
+
+
+    @Test
+    public void patchRemoveSubmissionCCLicense() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        Community community = CommunityBuilder.createCommunity(context)
+                                              .withName("Community")
+                                              .build();
+
+        Collection collection = CollectionBuilder.createCollection(context, community)
+                                                 .withName("Collection")
+                                                 .build();
+
+        WorkspaceItem workspaceItem = WorkspaceItemBuilder.createWorkspaceItem(context, collection)
+                                                          .withTitle("Workspace Item")
+                                                          .build();
+
+        context.restoreAuthSystemState();
+
+        String adminToken = getAuthToken(admin.getEmail(), password);
+
+        // First add a license and verify it is added
+        List<Operation> ops = new ArrayList<Operation>();
+        AddOperation addOperation = new AddOperation("/sections/cclicense/uri",
+                                                     "http://creativecommons.org/licenses/by-nc-sa/4.0/");
+
+        ops.add(addOperation);
+        String patchBody = getPatchContent(ops);
+
+
+        getClient(adminToken).perform(patch("/api/submission/workspaceitems/" + workspaceItem.getID())
+                                              .content(patchBody)
+                                              .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                             .andExpect(status().isOk())
+                             .andExpect(jsonPath("$.sections.cclicense", allOf(
+                                     hasJsonPath("$.uri", is("http://creativecommons.org/licenses/by-nc-sa/4.0/")),
+                                     hasJsonPath("$.rights",
+                                                 is("Attribution-NonCommercial-ShareAlike 4.0 International")),
+                                     hasJsonPath("$.file.name", is("license_rdf"))
+                             )));
+
+
+        // Remove the license again and verify it is removed
+
+        List<Operation> removeOps = new ArrayList<Operation>();
+        RemoveOperation removeOperation = new RemoveOperation("/sections/cclicense/uri");
+
+        removeOps.add(removeOperation);
+        String removePatch = getPatchContent(removeOps);
+
+
+        getClient(adminToken).perform(patch("/api/submission/workspaceitems/" + workspaceItem.getID())
+                                              .content(removePatch)
+                                              .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                             .andExpect(status().isOk())
+                             .andExpect(jsonPath("$.sections", not(hasJsonPath("cclicense"))));
```

Comment:
Could we add a second test here to verify what happens if you attempt to Remove a non-existing license?